src/fsinfo.ts

```diff
@@ -0,0 +1,139 @@
+"use strict";
+
+import cockpit from "cockpit";
```

Comment:
Just to get everyone on the same page, this some day should live in `pkg/lib` in Cockpit right?